server/test/unit/BraveAlerterConfiguratorTest/getReturnMessageToOtherResponderPhoneNumbersTest.js

```diff
@@ -6,83 +6,104 @@ const { before, describe, it } = require('mocha')
   })
 
-  it('should get message when STARTED => WAITING_FOR_REPLY', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToOtherResponderPhoneNumbers(
-      'en',
-      CHATBOT_STATE.STARTED,
-      CHATBOT_STATE.WAITING_FOR_REPLY,
-    )
-
-    expect(returnMessage).to.equal(null)
-  })
-
-  it('should get message when STARTED => WAITING_FOR_CATEGORY', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToOtherResponderPhoneNumbers(
-      'en',
-      CHATBOT_STATE.STARTED,
-      CHATBOT_STATE.WAITING_FOR_CATEGORY,
-    )
-
-    expect(returnMessage).to.equal(`Another Responder has acknowledged this request. (You don't need to respond to this message.)`)
-  })
-
-  it('should get message when WAITING_FOR_REPLY => WAITING_FOR_CATEGORY', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToOtherResponderPhoneNumbers(
-      'en',
-      CHATBOT_STATE.WAITING_FOR_REPLY,
-      CHATBOT_STATE.WAITING_FOR_CATEGORY,
-      'Selected Category',
-    )
-
-    expect(returnMessage).to.equal(`Another Responder has acknowledged this request. (You don't need to respond to this message.)`)
-  })
-
-  it('should get no message when WAITING_FOR_CATEGORY => WAITING_FOR_CATEGORY', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToOtherResponderPhoneNumbers(
-      'en',
-      CHATBOT_STATE.WAITING_FOR_CATEGORY,
-      CHATBOT_STATE.WAITING_FOR_CATEGORY,
-    )
-
-    expect(returnMessage).to.equal(null)
-  })
-
-  it('should get message when WAITING_FOR_CATEGORY => COMPLETED', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToOtherResponderPhoneNumbers(
-      'en',
-      CHATBOT_STATE.WAITING_FOR_CATEGORY,
-      CHATBOT_STATE.COMPLETED,
-      'Selected Category',
-    )
-
-    expect(returnMessage).to.equal(
-      `The incident was categorized as Selected Category.\n\nThank you. This session is now complete. (You don't need to respond to this message.)`,
-    )
-  })
-
-  it('should get no message when COMPLETED => COMPLETED', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToOtherResponderPhoneNumbers(
-      'en',
-      CHATBOT_STATE.COMPLETED,
-      CHATBOT_STATE.COMPLETED,
-      ['Cat0'],
-    )
-
-    expect(returnMessage).to.equal(null)
-  })
-
-  it('should get default message if given something funky', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToOtherResponderPhoneNumbers('en', 'something funky', CHATBOT_STATE.COMPLETED, [
-      'Cat0',
-    ])
-
-    expect(returnMessage).to.equal('Error: No active session found')
+  languages.forEach(language => {
+    it('should get message when STARTED => WAITING_FOR_REPLY', () => {
+      const returnMessage = this.alertStateMachine.getReturnMessageToOtherResponderPhoneNumbers(
+        language,
+        CHATBOT_STATE.STARTED,
+        CHATBOT_STATE.WAITING_FOR_REPLY,
+      )
+      expect(returnMessage).to.equal(null)
+    })
```

Comment:
Oh I've never seen a test that starts with `forEach` loop - is this common practice? I guess it makes me wonder if we need to have all the tests run in english and in Spanish? or if we just need a couple tests to confirm the Spanish functionality.

For example do we need to run this test twice `should get message when STARTED => WAITING_FOR_REPLY'` when we are just checking that return message is null?